extensions.json

```diff
@@ -818,23 +818,22 @@
     {
       "id": "ms-vscode.js-debug",
       "repository": "https://github.com/microsoft/vscode-js-debug",
-      "version": "1.50.0",
-      "checkout": "v1.50.0",
-      "prepublish": "npm run compile && npm run package",
-      "extensionFile": "dist/js-debug.vsix"
+      "version": "1.50.1",
+      "checkout": "master",
+      "prepublish": "npm run compile && npm run package"
     },
     {
       "id": "ms-vscode.node-debug",
       "repository": "https://github.com/microsoft/vscode-node-debug",
-      "version": "1.44.8",
-      "checkout": "v1.44.8",
+      "version": "1.44.14",
+      "checkout": "master",
       "prepublish": "yarn prepublish && yarn package"
     },
     {
       "id": "mtxr.sqltools",
       "repository": "https://github.com/mtxr/vscode-sqltools",
-      "version": "0.23.0",
-      "checkout": "v0.23.0"
+      "version": "1.0.0",
+      "checkout": "master"
```

Comment:
This extension has very confusing release tags.

The latest version published to the Microsoft marketplace is `0.23.0`: https://marketplace.visualstudio.com/items?itemName=mtxr.sqltools

However, there is no `0.23.0` or `v0.23.0` tags in the repo. There is `v0.22.11`, which was last modified in July, and also `v1.2.0` which was last modified in May?

I have no idea what's going on here. Probably safer to just undo this upgrade.

Reply:
Done!